packages/manager/modules/billing/src/payment/method/add/controller.js

```diff
@@ -13,13 +13,15 @@ export default class BillingPaymentMethodAddCtrl {
     ovhPaymentMethod,
     OVH_PAYMENT_METHOD_TYPE,
     OVH_PAYMENT_METHOD_INTEGRATION_TYPE,
+    coreConfig,
   ) {
     this.$translate = $translate;
     this.Alerter = Alerter;
     this.ovhContacts = ovhContacts;
     this.ovhPaymentMethod = ovhPaymentMethod;
     this.OVH_PAYMENT_METHOD_TYPE = OVH_PAYMENT_METHOD_TYPE;
     this.OVH_PAYMENT_METHOD_INTEGRATION_TYPE = OVH_PAYMENT_METHOD_INTEGRATION_TYPE;
+    this.user = coreConfig.getUser();
```

Comment:
Regarding my suggestions on HTML templates:

```suggestion
    this.coreConfig= coreConfig;
```